website/content/docs/bloop-basics.md

```diff
@@ -63,65 +73,30 @@ Gradle or brand-new build tools like CBT and mill, you should benefit from
 the fastest of the workflows no matter what the internal architecture of your
 build tool is.
 
-### Components
-
-Bloop has two main components, a client and a server. If you're looking for
-ways to extend this model, head to the [Integration
-guide]({{< ref "integration-guide.md" >}}).
-
-#### `bloop`, the client
-
-The `bloop` client is the tool that you use to run any bloop command. The
-client is a python script that implements Nailgun's protocol and communicates
-with the server. It is a fast CLI tool that gives you immediate feedback from
-the server.
-
-#### `blp-server`, the server
-
-The server is called `blp-server`, and it's a long-running application that
-runs on the background and keeps the state of all your projects and compiler
-instances.
-
-When you run `bloop compile` with the client, the server receives the request
-and spawns a thread to compile your project, logging you back everything that
-the compiler outputs. This server accepts requests in parallel except for
-those tasks that are already running.
-
-
-
-### Other properties
-
 #### Runtime independent
 
 Test or run your Scala application on the JVM, Scala.js and Scala Native.
 Bloop interfaces with the API of every runtime that the Scala ecosystem
 supports so that external tools don't have to.
 
-#### Familiarity
+#### Command line friendly
 
-Bloop is not a long running application like the sbt shell. You run it, get
-the result, and you have your shell back. This is closer to the workflow that
-many developers love and are used to in other tools like Maven, Gradle and
-npm.
+The primary way to interact with bloop is directly from the command line
+instead of through a shell or repl. This workflow is more familiar to how
+most of developer tools work (for example, Maven, Gradle or sbt) and allows
+you to reuse the same terminal for other tasks.
 
 #### Simplicity
 
-You can think of bloop as a dummy end-user application that supports build
-core commands. Bloop does not require you to learn a complex DSL to interact
-with it, or a complicated configuration format. `--help` takes you a long
-way.
-
-#### Rolling release
-
-Bloop releases feature the latest of the advancements in compiler APIs. If
-you use it, you will be the first one to benefit from performance speedups,
-bugfixes and other changes occurring upstream in both [scala/scala] and
-[sbt/zinc].
+You can think of bloop as a simple, end-user application that runs Bloop's
```

Comment:
We should not take the word "simple" lightly IMO, any tool that claims to be "simple" should back it with solid arguments See https://www.infoq.com/presentations/Simple-Made-Easy A client-server architecture and multi-layered builds is far from simple according to many people's view on what "simple" means.

Alternative suggestions
* Easy to use
* Easy to learn

> "`--help` takes you a long way."

You might want to expand on this.

```
$ bloop --help
Usage: bloop [options] cmd arg1 arg2 ...

Options:
  -h, --help            show this help message and exit
  --nailgun-server=NAILGUN_SERVER
  --nailgun-port=NAILGUN_PORT
  --nailgun-filearg=NAILGUN_FILEARG
  --nailgun-showversion
  --nailgun-help
```


The actual help page is not helpful either.
```
$ bloop help
bloop-frontend 1.0.0-M4
Usage: bloop-frontend [options] [command] [command-options]


Available commands: about, bsp, clean, compile, configure, console, help, projects, run, test
Type `bloop-frontend 'command' --help` for help on an individual command
```